includes/admin/class-evf-admin-entries.php

```diff
@@ -317,6 +318,33 @@ public static function update_status( $entry_id, $status = 'publish' ) {
 
 		return $update;
 	}
+
+	/**
+	 * Check new entries with heartbeat API.
+	 *
+	 * @since 1.4.10
+	 *
+	 * @param  array  $response  The Heartbeat response.
+	 * @param  array  $data      The $_POST data sent.
+	 * @param  string $screen_id The screen id.
+	 * @return array The Heartbeat response.
+	 */
+	public function check_new_entries( $response, $data, $screen_id ) {
+		if ( 'everest-forms_page_evf-entries' === $screen_id ) {
+			$form_id       = ! empty( $data['evf_new_entries_form_id'] ) ? absint( $data['evf_new_entries_form_id'] ) : 0;
+			$last_entry_id = ! empty( $data['evf_new_entries_last_entry_id'] ) ? absint( $data['evf_new_entries_last_entry_id'] ) : 0;
+
+			// Count new entries.
+			$entries_count = evf_get_count_entries_by_last_entry( $form_id, $last_entry_id );
+
+			if ( ! empty( $entries_count ) ) {
```

Comment:
Suggestion: I think we should check even the else portion here and if possible on all of our code. I don't see a way to send form_id as 0 but if it is possible then it might throw an error.

Reply:
Ah since suggestion. Can you test for a new form, what is the last entry ID data attribute thrown?

Reply:
I just checked it and it was my mistake that I didn't see that the length of entry list was checked and if it isn't set then heartbeat won't start.